### DISCUSSION TITLE
Import data from Wakatime

### DISCUSSION THREAD

**Discussion by andre-paulo98** [muety/wakapi#298] [Q&A] [answered]
Hello, there are a few issues on this repo claiming that it's possible to import data from Wakatime, on readme it also mentions that it's possible, but I can't seem able to find the option to import it.
Am I missing something?
This is all I see:
![image](https://user-images.githubusercontent.com/19685105/147989052-70f30d4b-5dd0-4ca8-9883-0bc1a14ed886.png)

~~Also I was testing the hosted version on wakapi.dev and it seems somewhat slow to respond, is this normal, if I selfhost will it be the same?~~
Installed locally and it seems fine.

Thanks!

**Comment by muety** [comment, marked as answer]
After you add your WakaTime API key, you'll be given the opportunity to import heartbeats. Have some patience, the import may take a couple of minutes. Does this answer your question?

**Reply by andre-paulo98** [reply]
Oh, I was looking for an option to manually import. I looked at that option and I thought that it was only used to forward the requests, and not setup the importer.

**Reply by muety** [reply]
Yeah, the description is perhaps a bit confusing, I might want to change it to be more clear.

**Reply by andre-paulo98** [reply]
Just tested it and it works perfectly. Thank you!
